lib/kitchen/driver/openstack.rb

```diff
@@ -325,11 +320,11 @@ def parse_ips(pub, priv)
 
       def add_ohai_hint(state)
         info 'Adding OpenStack hint for ohai'
-        ssh = Fog::SSH.new(*build_ssh_args(state))
-        ssh.run([
-          %(sudo mkdir -p #{Ohai::Config[:hints_path][0]}),
-          %(sudo touch #{Ohai::Config[:hints_path][0]}/openstack.json)
-        ])
+        mkdir_cmd = "sudo mkdir -p #{Ohai::Config[:hints_path][0]}"
```

Comment:
I'd create a helper method for the hints path:

```ruby
def hints_path
  Ohai::Config[:hints_path][0]
end
```

... and use that here so you're not having to dissect an array twice.